src/aap_eda/settings/default.py

```diff
@@ -195,14 +195,23 @@ def _get_boolean(name: str, default=False) -> bool:
     "ansible_base.resource_registry",
     "ansible_base.jwt_consumer",
     "ansible_base.rest_filters",
+    "ansible_base.feature_flags",
     # Local apps
     "aap_eda.api",
     "aap_eda.core",
 ]
 
+
+def toggle_feature_flags(_flags, _settings):
```

Comment:
Why not just assign the value of `FLAGS` from this function constructor, following the same pattern that we already have for databases or redis? 
In this way we have a function that is not pure and modifies the global state which is not a good practice and source of potential issues.

Reply:
After syncing on Slack, we will leave this as is now, but can potentially iterate on this later